phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java

```diff
@@ -46,13 +47,17 @@
     private final String applicationName;
 
     /**
-     * Create the setupUi
-     *
-     * @param title title of the setupUi
+     * constructor
+     * @param title wizard title
+     * @param miniature miniature for the setup wizard (usually miniature of the application which shall be installed)
+     * @param messageSender
+     * @param setupUiFactory
      * @param userHome
+     * @param applicationUserRoot
```

Comment:
I'm especially interested in a description of the params that have no description yet^^
For example: what exactly is the `applicatonUserRoot`?

Reply:
I didn't know by heart so I didn't write anything. Also, I didn't change this code at all.